source/io/module_javaScriptInvoke.js

```diff
@@ -428,7 +428,7 @@ var assignJavaScriptInvoke;
             };
 
             var jsapi;
-            if (functionToCall.length === parameters.length + 1) {
+            if (isInternalFunction || functionToCall.length === parameters.length + 1) {
```

Comment:
Should there be some error if: (isInternalFunction && functionToCall.length != parameters.length + 1) ?

Reply:
Nope, the use case is that an internal function can use the `arguments` variable in a function so the same function can be used with a variable number of parameters.

ie:
```javascript
// myFunc is called by multiple JavaScriptInvoke calls. One calls with zero parameters, one calls with 1, one calls with 5, etc

window.myFunc = function () {
    for (var i = 0; i < arguments.length; i++) {
        const currArg = arguments[i];
        // do something with currArg
    }
}

```